tests/test_utils_misc/test_return_with_argument_inside_generator.py

```diff
@@ -12,6 +12,13 @@
 def _indentation_error(*args, **kwargs):
     raise IndentationError()
 
+def create_ast_return(value=None):
+    return (
+        unittest.mock.MagicMock(name='ast.Return', value=None)
```

Comment:
Hmm my point is that I expect this test to test real code that produces specific ast nodes when parsed.

Reply:
(hpefully that makes sense)

Reply:
Yes, that makes sense. I will push a new change tomorrow. Thank you for the feedback!